e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -23,13 +23,20 @@ const cromshellSubmitRCmd = (cromshell_version) =>
   `system2('${cromshell_version}', args = c('submit', ` +
   `'${wdlFileName}','${jsonFileName}'), stdout = TRUE, stderr = TRUE)`;
 
+const cromshellStatusPythonCmd = (cromshell_version, cromshellJobId) =>
```

Comment:
Please add types to the parameters for all of these functions.

Reply:
Done, added string